lib/ansible/playbook/role_include.py

```diff
@@ -79,8 +80,15 @@ def get_block_list(self, play=None, variable_manager=None, loader=None):
         ri = RoleInclude.load(self._role_name, play=myplay, variable_manager=variable_manager, loader=loader, collection_list=self.collections)
         ri.vars.update(self.vars)
 
+        if variable_manager is not None:
+            available_variables = variable_manager.get_vars(play=myplay, task=self)
+        else:
+            available_variables = {}
+        templar = Templar(loader=loader, variables=available_variables)
+        from_files = dict((k, templar.template(v)) for k, v in self._from_files.items())
```

Comment:
I think we can just do:
```suggestion
        from_files = templar.template(self._from_files)
```

Reply:
thanks, why did I overcomplicate that